### DISCUSSION TITLE
Blazorstrap Modal

### DISCUSSION THREAD

**Discussion by markstraw** [vikramlearning/blazorbootstrap-starter-templates#11] [Q&A] [answered]
I am running NET8.BlazorWebAppRenderModeServerGlobal 

I grabbed modal component source code to create a modal from https://demos.blazorbootstrap.com/modals

I can't work out how to get past this issue - Multiple components use the tag 'Modal' 

![net8modal](https://github.com/vikramlearning/blazorbootstrap-starter-templates/assets/3908763/d5181c9b-e092-448c-bdf3-95fee0fbca1f)

**Comment by gvreddy04** [comment, marked as answer]
@markstraw Thanks for trying **BlazorBootstrap**. There's a conflict between the new file name `Modal.razor` and the Modal component. Please rename the file `Modal.razor` to something else.
